src/shell/components/InviteMembersModal/index.tsx

```diff
@@ -108,29 +233,82 @@ const InviteMembersModal = ({ onClose }: Props) => {
             renderInput={(params) => (
               <TextField
                 {...params}
-                sx={{ ".MuiOutlinedInput-root ": { alignItems: "baseline" } }}
-                multiline
-                rows={3}
+                sx={{
+                  ".MuiOutlinedInput-root ": {
+                    alignItems: "baseline",
+                    minHeight: 93,
+                  },
+                }}
+                ref={autocompleteRef}
                 error={emailError}
-                placeholder="e.g. name@zesty.io"
-                helperText={emailError ? "Invalid email format" : " "}
+                placeholder={
+                  emails.length ? "" : "Email, comma or space separated"
+                }
+                helperText={
+                  emailError ? "Please enter a valid email address." : " "
+                }
                 onKeyDown={(event) => {
                   setEmailError(false);
-                  if (event.key === "Enter" || event.key === ",") {
-                    if (
-                      inputValue &&
-                      inputValue.match(/^[^@]+@[^@]+\.[^@]+$/)
-                    ) {
-                      setEmails([...new Set([...emails, inputValue])]);
-                      setInputValue("");
+                  if (
+                    event.key === "Enter" ||
+                    event.key === "," ||
+                    event.key === " "
+                  ) {
+                    if (inputValue && !inputValue.match(/^\s+$/)) {
+                      if (inputValue.trim().match(emailAddressRegexp)) {
+                        event.preventDefault();
+                        setEmails([...new Set([...emails, inputValue.trim()])]);
+                        setInputValue("");
+                      } else {
+                        setEmailError(true);
+                      }
                     } else {
-                      setEmailError(true);
+                      setEmailError(false);
                     }
                   }
+
+                  if (event.key === "Backspace" && !inputValue) {
+                    event.stopPropagation();
+
+                    // HACK: Needed to prevent the default behavior of autocomplete which autodeletes the right most tag on backspace
```

Comment:
@agalin920 
There is no visual reference present here, so I am assuming you are referring to the double backspace which is required to remove a chip. If so, then we are following this design pattern to prevent accidental deletes. This design pattern can be observed in Gmail (a material UI app) and Slack and is from where it is adopted. 

Using a single backspace is not the worst solution and we can move forward with using that for now if that is your recommendation.

Note: At the moment the old auto-complete chip inputs used by one to many and one to one fields are going to be phased out and replaced with a new type of input instead (similar to Media field). In the future though, if we do continue to use the autocomplete chip input being used frequently for other parts of Zesty, then we would want this double backspace to delete functionality to be present. And yes it would make sense to add it into the design system itself.